test/v3nametest.c

```diff
@@ -288,6 +288,9 @@ static int run_cert(X509 *crt, const char *nameincert,
         char *name = OPENSSL_malloc(namelen + 1);
         int match, ret;
 
+        if (!TEST_ptr(name)) {
+            return 0;
+        }
```

Comment:
Please remove the curly brackets when there is just a single statement in the if.